AnnService/inc/Core/BKT/ParameterDefinitionList.h

```diff
@@ -43,5 +43,8 @@ DefineBKTParameter(m_iThresholdOfNumberOfContinuousNoBetterPropagation, int, 3L,
 DefineBKTParameter(m_iNumberOfInitialDynamicPivots, int, 50L, "NumberOfInitialDynamicPivots")
 DefineBKTParameter(m_iNumberOfOtherDynamicPivots, int, 4L, "NumberOfOtherDynamicPivots")
 DefineBKTParameter(m_iHashTableExp, int, 4L, "HashTableExponent")
+DefineBKTParameter(m_pSamples.rowsInBlock, int, 1024 * 1024, "DataBlockSize")
+DefineBKTParameter(m_pGraph.RowsInBlock(), int, 1024 * 1024, "GraphBlockSize")
```

Comment:
could you change to m_pGraph.rowsInBlock to align style?

Reply:
The rowsInBlock is the member variable of Dataset, not the graph and the labelset.